.template/variants/web/app/javascript/build.js

```diff
@@ -6,6 +6,7 @@ require('esbuild')
     inject: ['app/javascript/global.js'],
     bundle: true,
     sourcemap: true,
```

Comment:
@sanG-github a small comment, we really don't use source map on Production, should we set this as 
```suggestion
    sourcemap: !watch,
```

Reply:
Wd do use `sourcemap` 😅 You might not but sourcemaps as essential for debugging

Reply:
@olivierobert , that's interesting, we have `--no-source-map` for `sass`, for all environments
https://github.com/nimblehq/rails-templates/blob/b913280bc0fc13f95d2406212297e4679c4d3601/.template/variants/web/package.json.rb#L37

Then, I tried enabling it for a local project, and it was trying to reference the file by its path 😄 

<img width="1287" alt="image" src="https://github.com/nimblehq/rails-templates/assets/39977805/21c3e90d-dfb2-4fc2-8173-13b21b75e5bd">

I searched around and it seems like an [existing issue](https://github.com/rails/cssbundling-rails/issues/68) for cssbundling-rails

I was wondering if you ever were able to use source maps for SCSS assets in a Rails project?

Reply:
@tyrro Nice catch! :o Should we create a dedicated issue to investigate this?

Reply:
@olivierobert but we don't debugging on Production 🤔

Reply:
ok, after checking this article https://css-tricks.com/should-i-use-source-maps-in-production/ it's better to have the sourcemap information in the Production build cc @tyrro @olivierobert

Reply:
@malparty if we have a chance, we could just remove this config on the same PR ;) 

![image](https://github.com/nimblehq/rails-templates/assets/11751745/7bcb6fa4-9932-4de2-9b4e-9b240f831232)

Reply:
@andyduong1920 ,  but `source-map` doesn't work for `cssbundling-rails` (please refer to my last comment).

So if we remove `--no-source-map` option, it'll try to generate the source map, which will result in the following:

https://github.com/nimblehq/rails-templates/assets/39977805/79f48f98-5316-4ca1-9a71-96eab772de9a

which is not very helpful for debugging 😄 

@malparty , IMHO it's not worth it at this moment.
`jsbundling-rails` support `source-map` (and we are using it too btw), `css-bunding-rails` yet to support it. So, I'd keep it as it is regarding the `--no-source-map` option!

Reply:
@andyduong1920 lets create an issue for that :)

Reply:
Issued created https://github.com/nimblehq/rails-templates/issues/414 @malparty @tyrro